scripts/us_census/cps_hvs/README.md

```diff
@@ -0,0 +1,47 @@
+# Importing FRED homeownership rate data into Data Commons
+
+Author: KilimAnnejaro
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Artifacts](#about-the-import)
+
+## About the dataset
+
+### Download URL
+
+The CSV file is available for download from <https://www.kaggle.com/census/homeownership-rate-time-series-collection>.
+
+### Overview
+
+This dataset contains two variables:
+1. date: The date when the value was measured.
```

Comment:
I don't think date is considered a variable, I would say there is just one variable here